packages/theme-check-docs-updater/src/themeLiquidDocsManager.ts

```diff
@@ -30,6 +31,10 @@ export class ThemeLiquidDocsManager implements ThemeDocset, JsonSchemaValidators
     return this.loadResource('tags', []);
   });
 
+  systemTranslations = memo(async (): Promise<Translations> => {
+    return this.loadResource('shopify_system_translations', []);
```

Comment:
I think we want the default value to be `{}` here?

Reply:
Great catch! Fixed :)